bokeh/themes/dark_lite.json

```diff
@@ -0,0 +1,66 @@
+{
+    "attrs": {
+        "Figure" : {
+            "background_fill_color": "#20262B",
+            "border_fill_color": "#15191C",
+            "outline_line_color": "#E0E0E0",

```

Comment:
JSON specification doesn't allow for trailing commas. Perhaps we use an extended (aka. non-compliant) parser, but still it would be good to have a *.json file contain valid JSON.

Reply:
You can run `python -m json.tool some_file.json` to verify if `some_file.json` parses correctly.